docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,158 @@
+
+## Requirements
+
+* Terraform CLI 1.4+
+* A running PingFederate server accessible over HTTPS, or Docker CLI to start one
+* If using Docker to start a PingFederate server, a DevOps license is required - [Register for the DevOps program here](https://devops.pingidentity.com/how-to/devopsRegistration/)
+
+## Start a PingFederate Docker image to be configured
+
+!!! warning "Using an Existing PingFederate Server"
+    If you already have a running PingFederate server that you can reach over HTTPS, you can skip this step. The provider can be used with any PingFederate server.
+
+First, start a PingFederate server using Docker. Your DevOps credentials will be read from the `${HOME}/.pingidentity/config` file. The HTTPS port (default `9999`) must be exposed.
+
+```shell
+docker run --name pingfederate_terraform_provider_container \
+  -d -p 9031:9031 \
+  -d -p 9999:9999 \
+  --env-file "${HOME}/.pingidentity/config" \
+  -e SERVER_PROFILE_URL=https://github.com/pingidentity/pingidentity-server-profiles.git \
+  -e SERVER_PROFILE_BRANCH=terraform-provider-pingfederate-1125 \
+  -e SERVER_PROFILE_PATH=terraform-provider-pingfederate/pingfederate \
+  pingidentity/pingfederate:$${PINGFEDERATE_PROVIDER_PRODUCT_VERSION:-12.0.0}-latest
+```
+
+After starting the container, follow the logs until the server becomes available.
+
+```shell
+docker logs -f pingfederate_terraform_provider_container
+```
+
+After you see the following message in the container logs, the server is ready to receive requests from the provider:
+
+```shell
+PingFederate is up
+```
+
+## Accessing the PingFederate API
+
+Using the port mappings from the example above, PingFederate will be available at `https://localhost:9999/pingfederate/app#/`.
+
+If you want to access the API documentation, it is available at `https://localhost:9999/pf-admin-api/api-docs/#/`.
+
+The PingFederate Terraform provider applies configuration at the API endpoints using the `9999` port over HTTPS when accessing the product locally under Docker.
+
+## Determine credentials that are able to configure the server
+
+The configuration API connection established by the provider uses basic authentication. The provider will need the username and password of a user that has permissions to manage server configuration.
+
+In the PingFederate Docker image, the default administrative user has the username `administrator` with password `2FederateM0re`.
+
+## Determine what version of PingFederate you are running
+
+The provider requires that the version of PingFederate is specified.  
+
+You can do this one of two ways:
+
+* Using the `product_version` attribute when configuring the provider:
+
+```shell
+provider "pingfederate" {
+  ...
+  product_version = "12.0.0"
```

Comment:
why do I not see this field on the terraform registry docs? recent add?

Reply:
Yeah I forgot to add this one to the docs. I just merged in a fix for that today, so it'll be on the registry docs after the next provider release.